lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -68,7 +68,7 @@ class OffscreenImages extends ByteEfficiencyAudit {
    * @return {?Object}
    */
   static computeWaste(image, viewportDimensions) {
-    const url = URL.getDisplayName(image.src, {preserveQuery: true});
+    const url = URL.getURLDisplayName(image.src, {preserveQuery: true});
```

Comment:
Oh I was just suggesting the name `getURLDispalyName` on the Util object not changing the name on `URL` haha